auth/access.js

```diff
@@ -0,0 +1,36 @@
+const config = require('./../config');
+const jwt = require('jsonwebtoken');
+const { crudService, errorService } = require('./../services');
+const { UserModel } = require('./../models');
+const { URLS } = require('./../shared/constants');
+
+const access = (req, res, next) => {
+    try {
+        const token = req.headers.authorization.split(' ')[1];
```

Comment:
what if req.headers.authorization is undefined ?

Reply:
I was doing that. So in this case we catch error with message ("Cannot read property 'split' of undefined") and we run business logic of 'catch' block. So everything is working.

Reply:
Its not good idea to rely on such exceptions and build bussines logic on these.
Any time, If its possible, you should prevent exceptions provided by system, but throw your own.
Its should be strong approach to control states of varaibles, otherwise its threatening to have unexpected behavior, which sometimes really hard to reproduce.
So if we are not sure that `req.headers.authorization` exists, we should check it first.

Reply:
OK. I'll do that.